_includes/gc-follow-us.html

```diff
@@ -1,9 +1,17 @@
 <section class="provisional gc-followus">
 	<h2>
-		{%- if i18nText-lang == "fr" -%}
-			Dans les médias sociaux
+		{%- if include.inlineLayout -%}
+			{%- if page.language == "fr" -%}
```

Comment:
Actually use `i18nText-lang` instead of `page.language` everywhere in this file. This variable is made available from the **variable-core** include you will add on the `institutional-landing-en` and `fr` pages.